libfwupdplugin/fu-usb-device-private.h

```diff
@@ -8,5 +8,7 @@
 
 #include "fu-usb-device.h"
 
+#define FU_USB_DEVICE_EMULATION_TAG "interesting"
```

Comment:
@superm1 I'm not sure about this name; maybe `org.freedesktop.fwupd.emulation.v1` might be more future proof, although it's a lo more wordy. I'm trying to convey that it's interesting to the daemon.

Reply:
I think the longer string makes sense